.env.default

```diff
@@ -1,8 +1,8 @@
 # Private key must be prepended by '0x'
-DEPLOYMENT_PRIVATE_KEY="" 
+DEPLOYMENT_PRIVATE_KEY="0xf2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e0164837257d" 
```

Comment:
Did we mean to commit this?

Reply:
Yes, the default generally has sample values. I'm taking the first pair produced when initializing chain.